<!-- -->
<launch>
  <arg name="role_name" default="ego_vehicle" />
  <arg name="detect_range" default="60.0" />
  <arg name="publish_object_cube" default="true" />
  <arg name="publish_object_info" default="true" />

  <include file="$(find message_translator)/launch/message_translator_core.launch" >
    <arg name='role_name' value='$(arg role_name)'/>
    <arg name='detect_range' value='$(arg detect_range)'/>
  </include>

  <include file="$(find message_translator)/launch/detected_object_publisher.launch" >
    <arg name='publish_object_cube' value='$(arg publish_object_cube)'/>
    <arg name='publish_object_info' value='$(arg publish_object_info)'/>
  </include>
</launch>
